docs/eventing/samples/choice/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Choice Example"
+linkTitle: "Choice"
+weight: 10
+type: "docs"
+---
```

Comment:
```suggestion
---

{{% readfile file="README.md" relative="true" markdown="true" %}}
```

Reply:
This pulls the content from the README into the index file, so that it display properly on the site.

Reply:
Oh, I see you have these everywhere else they're needed; looks like this one just got skipped.